src/zeit/solr/converter.py

```diff
@@ -274,6 +275,24 @@ def process(self, value, doc_node):
         self.append_to_node(unicode(type), doc_node)
 
 
+class VolumeIndex(Index):
+    # XXX kludgy special case to only apply to IVolume objects and not to other
```

Comment:
Das nächste Mal schreib ich in die PR-Beschreibung mit rein, dass ich absichtlich entschieden habe, die Mechanik so zu lassen, auch wenn sie doof ist (und nicht das neuere schönere tollere von zeit.retresco einzuziehen), weil das zeit.solr ne Halbwertszeit von <6 Monaten hat und sich das IMHO nicht mehr lohnt.